docs/spec.md

```diff
@@ -6138,6 +6200,18 @@ def dequantize_op_quantize(op, *inputs_and_output_type):
   float_result = op(*float_inputs)
   rounded_result = round_nearest_even(float_result / scale(output_type))
   return clamp(storage_min(output_type), rounded_result, storage_max(output_type))
+
+def dequantize_select_quantize(pred, on_true, on_false, output_type):
+  float_on_true = (on_true - zero_point(on_true)) * scale(on_true)
+  float_on_false = (on_false - zero_point(on_false)) * scale(on_false)
+  float_result = select(pred, float_on_true, float_on_false)
+  rounded_result = round_nearest_even(float_result / scale(output_type))
+  return clamp(storage_min(output_type), rounded_result, storage_max(output_type))
+
+def dequantize_compare(lhs, rhs, comparison_direction):
+  float_lhs = (lhs - zero_point(lhs)) * scale(lhs)
+  float_rhs = (rhs - zero_point(rhs)) * scale(rhs)
+  return compare(float_lhs, float_rhs, comparison_direction, FLOAT)
 ```
```

Comment:
Just a reminder so that we don't forget. We'd want to use `quantize` and `dequantize` from #1496 in these new definitions as well.

Reply:
done.